Failing tests on master

```
=== RUN   TestBinary
521
--- FAIL: TestBinary (0.40s)
522
    require.go:794: 
523
        	Error Trace:	binary_test.go:54
524
        	Error:      	Received unexpected error:
525
        	            	Version 'v0.18.0' not found
526
        	            	
527
        	            	github.com/src-d/regression-core.(*Releases).Get
528
        	            		/home/travis/gopath/src/github.com/src-d/regression-core/releases.go:71
529
        	            	github.com/src-d/regression-core.(*Binary).downloadRelease
530
        	            		/home/travis/gopath/src/github.com/src-d/regression-core/binary.go:133
531
        	            	github.com/src-d/regression-core.(*Binary).Download
532
        	            		/home/travis/gopath/src/github.com/src-d/regression-core/binary.go:111
533
        	            	github.com/src-d/regression-core.TestBinary
534
        	            		/home/travis/gopath/src/github.com/src-d/regression-core/binary_test.go:47
535
        	            	testing.tRunner
536
        	            		/home/travis/.gimme/versions/go1.11.12.linux.amd64/src/testing/testing.go:827
537
        	            	runtime.goexit
538
        	            		/home/travis/.gimme/versions/go1.11.12.linux.amd64/src/runtime/asm_amd64.s:1333
539
        	Test:       	TestBinary
```

--- a/.gitignore
+++ b/.gitignore
@@ -34,3 +34,4 @@ coverage.txt
 bin/
 build/
 
+vendor

--- a/binary_test.go
+++ b/binary_test.go
@@ -26,9 +26,9 @@ func TestBinary(t *testing.T) {
 		ok       bool
 	}{
 		{
-			version:  "v0.18.0",
-			binSize:  39535564,
-			yamlSize: 3936,
+			version:  "v0.23.1",
+			binSize:  42178428,
+			yamlSize: 5007,
 			ok:       true,
 		},
 		{

